### PR TITLE
Entitlement Management Extensibility GA

### PR DESCRIPTION
**Pull request by OWinfreyATL**
This docs PR is tracked via this work item https://dev.azure.com/msft-skilling/Content/_workitems/edit/98618

### CODE REVIEW

**Review by msewaweru — CHANGES_REQUESTED** [review]
@OWinfreyATL - thanks for the work on this PR. I have left a few review comments including the below:
1. Reconcile the properties of the accessPackageAssignmentRequestCallbackData resource type between the schema and the API.md.
2. For both accessPackageAssignmentWorkflowExtension  and AccessPackageAssignmentRequestWorkflowExtension check the list of properties for a missing inherited property 'clientconfiguration' and 'type' that is defined in the API.md and not in the schema.

File: api-reference/v1.0/resources/accesspackageassignmentrequestworkflowextension.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+|callbackConfiguration|[customExtensionCallbackConfiguration](../resources/customextensioncallbackconfiguration.md)|The callback configuration for a custom extension.|
+|createdBy|String|The userPrincipalName of the user or identity of the subject that created this resource. Read-only.|
+|createdDateTime|DateTimeOffset|When the object was created.|
+|description|String|Description for the customAccessPackageWorkflowExtension object. Inherited from [customCalloutExtension](../resources/customcalloutextension.md).|
+|displayName|String|Display name for the customAccessPackageWorkflowExtension object. Inherited from [customCalloutExtension](../resources/customcalloutextension.md).|
+|endpointConfiguration|[customExtensionEndpointConfiguration](../resources/customextensionendpointconfiguration.md)|The type and details for configuring the endpoint to call the logic app's workflow. Inherited from [customCalloutExtension](../resources/customcalloutextension.md).|
+|id|String|Read-only.|
+|lastModifiedBy|String|The userPrincipalName of the identity that last modified the object. |
+|lastModifiedDateTime|DateTimeOffset|When the object was last modified.|
```
[the remark this answers is not in the document]
Comment (OWinfreyATL) [reply api-reference/v1.0/resources/accesspackageassignmentrequestworkflowextension.md]: Confirmed that the "type" property is an oversight by engineering. clientconfiguration added.

File: api-reference/v1.0/resources/accesspackageassignmentrequestcallbackdata.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Access package assignment request workflow calllback back that custom extension endpoint callback, derived from [customextensiondata](../resources/customextensiondata.md)
+
+Inherits from [customExtensionData](../resources/customextensiondata.md).
+
+## Properties
+
+|Property|Type|Description|
+|:---|:---|:---|
+|customExtensionStageInstanceId|String|Unique identifier of the callout to the custom extension.|
+|customExtensionStageInstanceDetail|String|Details for the callback.|
```
[the remark this answers is not in the document]
Comment (OWinfreyATL) [reply api-reference/v1.0/resources/accesspackageassignmentrequestcallbackdata.md:23]: Original schema update shows these two: https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/8235696?_a=files

File: api-reference/v1.0/resources/logicapptriggerendpointconfiguration.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+The configuration details for the logic app's endpoint that is associated with a custom access package workflow extension. Derived from the [customExtensionEndpointConfiguration](customextensionendpointconfiguration.md) abstract type.
+
+## Properties
+
+|Property|Type|Description|
+|:---|:---|:---| 
+|logicAppWorkflowName|String|The name of the logic app.|
+|resourceGroupName|String|The Azure resource group name for the logic app.|
+|subscriptionId|String|Identifier of the Azure subscription for the logic app.|
+|url|String| The URL to the logic app endpoint that will be triggered. Only required for app-only token scenarios where app is creating a [customCalloutExtension](../resources/customcalloutextension.md) without a signed-in user.|
```
[the remark this answers is not in the document]
Comment (OWinfreyATL) [reply api-reference/v1.0/resources/logicapptriggerendpointconfiguration.md:23]: Schema shows these 4 properties: https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/8235696?_a=files

File: api-reference/v1.0/resources/accesspackageassignmentrequestcallbackdata.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (msewaweru) [reply api-reference/v1.0/resources/accesspackageassignmentrequestcallbackdata.md:23]: > Original schema update shows these two: https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/8235696?_a=files

The API.md then needs to be updated to reflect what's in the schema.

File: changelog/Microsoft.IGAELM.json [hunk 4]
```diff
@@ -376,6 +376,176 @@
...
+          "Target": "logicAppTriggerEndpointConfiguration"
+        },
+        {
+          "Id": "e4f06e06-e395-4532-a745-2c8aa89db1b4",
+          "ApiChange": "Property",
+          "ChangedApiName": "customExtensionCalloutInstances",
+          "ChangeType": "Addition",
+          "Description": "Added the `customExtensionCalloutInstances` property to the [accessPackageAssignment](https://learn.microsoft.com/en-us/graph/api/resources/accessPackageAssignment?view=graph-rest-1.0) resource.",
+          "Target": "accessPackageAssignment"
+        },
```
Comment (msewaweru) [inline changelog/Microsoft.IGAELM.json:476]: This change is not capture in this PR.

File: changelog/Microsoft.IGAELM.json [hunk 5]
```diff
@@ -376,6 +376,176 @@
...
+          "Description": "Added the **customExtensionStageSettings** relationship to the [accessPackageAssignmentPolicy](https://learn.microsoft.com/en-us/graph/api/resources/accessPackageAssignmentPolicy?view=graph-rest-1.0) resource.",
+          "Target": "accessPackageAssignmentPolicy"
+        },
+        {
+          "Id": "e4f06e06-e395-4532-a745-2c8aa89db1b4",
+          "ApiChange": "Property",
+          "ChangedApiName": "customExtensionCalloutInstances",
+          "ChangeType": "Addition",
+          "Description": "Added the `customExtensionCalloutInstances` property to the [accessPackageAssignmentRequest](https://learn.microsoft.com/en-us/graph/api/resources/accessPackageAssignmentRequest?view=graph-rest-1.0) resource.",
+          "Target": "accessPackageAssignmentRequest"
```
Comment (msewaweru) [inline changelog/Microsoft.IGAELM.json:491]: This change is not capture in this PR.

File: changelog/Microsoft.IGAELM.json [hunk 6]
```diff
@@ -376,6 +376,176 @@
...
+          "Description": "Added the `customExtensionCalloutInstances` property to the [accessPackageAssignment](https://learn.microsoft.com/en-us/graph/api/resources/accessPackageAssignment?view=graph-rest-1.0) resource.",
+          "Target": "accessPackageAssignment"
+        },
+        {
+          "Id": "e4f06e06-e395-4532-a745-2c8aa89db1b4",
+          "ApiChange": "Resource",
+          "ChangedApiName": "customExtensionStageSettings",
+          "ChangeType": "Addition",
+          "Description": "Added the **customExtensionStageSettings** relationship to the [accessPackageAssignmentPolicy](https://learn.microsoft.com/en-us/graph/api/resources/accessPackageAssignmentPolicy?view=graph-rest-1.0) resource.",
+          "Target": "accessPackageAssignmentPolicy"
```
Comment (msewaweru) [inline changelog/Microsoft.IGAELM.json]: This change is not capture in this PR.

File: changelog/Microsoft.IGAELM.json [hunk 7]
```diff
@@ -376,6 +376,176 @@
...
+          "Description": "Added the [accessPackageAssignmentWorkflowExtension](https://learn.microsoft.com/en-us/graph/api/resources/accessPackageAssignmentWorkflowExtension?view=graph-rest-1.0) resource",
+          "Target": "accessPackageAssignmentWorkflowExtension"
+        },
+        {
+          "Id": "e4f06e06-e395-4532-a745-2c8aa89db1b4",
+          "ApiChange": "Resource",
+          "ChangedApiName": "accessPackageCustomWorkflowExtensions",
+          "ChangeType": "Addition",
+          "Description": "Added the **accessPackageCustomWorkflowExtensions** relationship to the [accessPackageCatalog](https://learn.microsoft.com/en-us/graph/api/resources/accessPackageCatalog?view=graph-rest-1.0) resource.",
+          "Target": "accessPackageCatalog"
```
Comment (msewaweru) [inline changelog/Microsoft.IGAELM.json]: This change is not capture in this PR.

**Review by msewaweru — CHANGES_REQUESTED** [review]
@OWinfreyATL - thanks for addressing the feedback provided. I have completed reviewing the changelog and there are a few additions that are not part of this PR.

Also note that we cannot mark this PR as content review complete until [this PR](https://github.com/microsoftgraph/microsoft-graph-docs/pull/20747), that deprecates the previous version of the extensibility APIs in beta, is merged.

File: api-reference/v1.0/resources/customextensiondata.md [hunk 8]
```diff
@@ -13,6 +13,7 @@ Namespace: microsoft.graph
 
 This is an abstract type noting the data being sent and received as part of extensibility scenarios based on [custom callout extensions](../resources/customcalloutextension.md). The following types are derived from this abstract type:
 
+- [accessPackageAssignmentRequestCallbackData](../resources/accesspackageassignmentrequestcallbackdata.md)
 - [customTaskExtensionCalloutData](../resources/identitygovernance-customtaskextensioncalloutdata.md)
 - [customTaskExtensionCallbackData](../resources/identitygovernance-customtaskextensioncallbackdata.md)
```
Comment (msewaweru) [inline api-reference/v1.0/resources/customextensiondata.md:18]: These 2 resources are not captured in the schema to be derived from customExtensionData.

File: changelog/Microsoft.IGAELM.json (hunk 4 above)
Comment (msewaweru) [reply changelog/Microsoft.IGAELM.json:476]: @OWinfreyATL - this has not been addressed yet.

File: changelog/Microsoft.IGAELM.json (hunk 5 above)
Comment (msewaweru) [reply changelog/Microsoft.IGAELM.json:491]: This too.

**Review by msewaweru — CHANGES_REQUESTED** [review]
@OWinfreyATL - I have a had another review. You missed 2 issues I had raised in the change log where the changes indicated are not in this PR. Please recheck. Thanks.

**Review by msewaweru — COMMENTED** [review]
@OWinfreyATL - I have completed my review of this PR and it looks good.

The approval of this PR is blocked by the [PR deprecating some custom extension modules](https://github.com/microsoftgraph/microsoft-graph-docs/pull/20747). Faith is reviewing it and will notify, so we can proceed to merge this work. Thanks.

**Review by msewaweru — APPROVED** [review]
@OWinfreyATL - This PR has been unblocked. Thanks for your patience.

**Pull request merged**